recipes/django-versatileimagefield/meta.yaml

```diff
@@ -0,0 +1,49 @@
+
+{% set name = "django-versatileimagefield" %}
+{% set version = "1.7.1" %}
+{% set sha256 = "3b15425d37b9d74a1b4087836d41624a56617a700bd4c95c0f228da60e0cbc3a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
```

Comment:
Is `pillow` a run requirement?

https://github.com/respondcreate/django-versatileimagefield/blob/master/setup.py#L18